docs/pages/enroll-resources/server-access/guides/encrypted-session-recordings/encrypted-session-recordings.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Encrypted Session Recordings
+description: How to enable encrypted session recordings using your CA key backend.
+labels:
+  - how-to
+  - zero-trust
+---
+
+Encrypted session recordings allow Teleport users to enable at-rest encryption
+of their session recording data. This guide explains how to setup encrypted
+session recordings.
+
+## How it works
+
+When encrypted recordings are enabled, Teleport will encrypt session recording
+data before saving it to disk or long term storage. Encryption keys are
+provisioned using the same key storage backend configured for CAs defined by
+the `ca_key_params` section of the Teleport Auth Service configuration file. In
+`node-sync` and `proxy-sync` session recording modes, session recording events
+are sent directly to the Teleport Auth Service where they are encrypted prior
+to being written to long term storage. In `node` and `proxy` session recording
+modes, session recording events are encrypted locally at the Teleport-protected
+server or Proxy Service instance before being written to disk for future upload
+to the Auth Service.
+
+Decryption of session recordings for replay is always facilitated by the
+Teleport Auth Service regardless of the session recording mode. This means
+replays can be viewed using the Teleport Web UI, or by using `tsh play`
+with a valid session ID. Replaying session recording files directly using
+`tsh play` is not possible for encrypted files.
+
+## Step 1/2. Configure Teleport
+
+In this section, you will configure Teleport to encrypt session recordings
+using the configured key storage backend.
+
+### Enable recording encryption
+
+Encrypted session recordings can be enabled through the Teleport Auth
+Service configuration file.
+
+You can edit the Auth Service configuration file as follows:
+
+```yaml
+# snippet from teleport.yaml
+auth_service:
+  session_recording_config:
+    encryption:
+      enabled: yes
+```
+
+### Optional step: Configure key storage backend
+
+Session recording encryption keys are provisioned using the same key storage
+backend configured for CAs. Software keys stored in Teleport's backend storage
+are the default, but other backends can also be configured:
+
+- [AWS KMS](../../../../zero-trust-access/deploy-a-cluster/aws-kms.mdx)
+- [Google Cloud KMS](../../../../zero-trust-access/deploy-a-cluster/gcp-kms.mdx)
+- [HSM](../../../../zero-trust-access/deploy-a-cluster/hsm.mdx)
+
+It is important that all Teleport Auth Service instances have access to the
+same keys and key storage backend to ensure all recorded sessions are always
```

Comment:
Can we re-write this to make it 100% clear that every access graph replica MUST access the same exact encryption key? This means sync HSM PK material or make a network HSM